examples/AzureSDKDemoSwiftUI/AzureSDKDemoSwiftUI/Utils/Common.swift

```diff
@@ -0,0 +1,183 @@
+        if AppState.internalBlobClient == nil {
+            guard let _ = AppState.application else {
+                fatalError("Application is not initialized. Unable to create Blob Storage Client.")
+            }
+            
+            let credential = StorageSASCredential(staticCredential: AppConstants.sasConnectionString)
+            AppState.internalBlobClient = try? StorageBlobClient(
+                endpoint: URL(string: "https://iosdemostorage1.blob.core.windows.net/")!,
+                credential: credential)
+        }
+        
+        guard AppState.internalBlobClient != nil else {
+            throw error
+        }
+
+        let client = AppState.internalBlobClient!
+        client.delegate = delegate
+        StorageBlobClient.maxConcurrentTransfers = 4
+        return client
+    }
+}
+
+class ActivtyViewController: UIViewController {
+    internal var spinner = UIActivityIndicatorView(style: .medium)
+
+    override func loadView() {
+        view = UIView()
+        view.backgroundColor = UIColor(white: 0, alpha: 0.7)
+
+        spinner.translatesAutoresizingMaskIntoConstraints = false
+        spinner.startAnimating()
+        view.addSubview(spinner)
+
+        spinner.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        spinner.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+    }
+}
+
+extension UIViewController {
+    internal func showAlert(error: Error?) {
+        guard presentedViewController == nil else { return }
+        var errorString: String
+        if let err = error {
+            switch error {
+            case let azureError as AzureError:
+                errorString = azureError.message
+            default:
+                let errorInfo = (err as NSError).userInfo
+                errorString = errorInfo[NSDebugDescriptionErrorKey] as? String ?? err.localizedDescription
+            }
+        } else {
+            errorString = "An error occurred."
+        }
+        let alertController = UIAlertController(title: "Error!", message: errorString, preferredStyle: .alert)
+        let title = NSAttributedString(string: "Error!", attributes: [
+            NSAttributedString.Key.foregroundColor: UIColor.red
+        ])
+        alertController.setValue(title, forKey: "attributedTitle")
+        let defaultAction = UIAlertAction(title: "OK", style: .default, handler: nil)
+        alertController.addAction(defaultAction)
+        present(alertController, animated: true)
+    }
+
+    internal func showAlert(message: String) {
+        guard presentedViewController == nil else { return }
+        let alertController = UIAlertController(title: "Blob Contents", message: message, preferredStyle: .alert)
+        let defaultAction = UIAlertAction(title: "Close", style: .default, handler: nil)
+        alertController.addAction(defaultAction)
+        present(alertController, animated: true)
+    }
+}
```

Comment:
Hopefully this can be repurposed into something easy to display a quick SwiftUI alert view.

Reply:
I actually copied a lot of this from the original demo project. I will add it to my to do to clean this up and remove anything that is unused.